selfdrive/car/mazda/carcontroller.py

```diff
@@ -23,17 +23,17 @@ def update(self, enabled, CS, frame, actuators):
                                                   CS.out.steeringTorque, SteerLimitParams)
       self.steer_rate_limited = new_steer != apply_steer
 
-      if CS.out.standstill and frame % 50 == 0:
+      if CS.out.standstill and frame % 20 == 0:
```

Comment:
Can you use DT_CTRL instead?

Reply:
not familiar with it pointer?

Reply:
I'm going to leave this as is for now @adeebshihadeh as it works fine now. DT_CTRL will require more code changes and more test drives which delay things if we want to get these fixes in 0.7.6.